docs/future_plans.md

```diff
@@ -0,0 +1,62 @@
+**Last updated:** January 2021.
+
+**Tl;dr:** We'll be redesigning how Stardoc works, and deprioritizing feature
+requests and minor bugs until that work is complete (targeting 2021 Q2).
+
+## Technical motivation
+
+Stardoc is currently the recommended tool for generating documentation of
+Starlark rules. It [replaces](skydoc_deprecation.md) Skydoc, the previous tool,
+which worked by mocking evaluation of .bzl files as Python code.
+
+Mocking is an inherently problematic approach for two reasons:
+
+1. It creates a maintenance burden for the tool maintainer (us). We have to
+ensure that the mocked definitions stay up-to-date as Bazel changes. These
+include not just `rule()` and `provider()`, but also a number of other symbols
+that don't directly affect documentation but still require stubs.
+
+2. It puts a constraint on the user: All of their documented .bzl files, as well
+as all of the .bzl dependencies they transitively load, must be compatible with
+the mock evaluation. This means users must be vigilant about writing Starlark
+code that lies in the intersection of what is understood as valid by Bazel and
+by Stardoc.
+
+Skydoc experienced an extreme version of this problem because it didn't even
```

Comment:
The sky* and star* prefixes always seem equivalent in my head. Perhaps emphasize the distinction:

"The Python-based skydoc experienced..."
"However, the Java-based stardoc also...

Reply:
Done, and italicized "Sky" in its first appearance.